src/main/java/com/esri/samples/raster/raster_rendering_rule/RasterRenderingRuleSample.java

```diff
@@ -0,0 +1,204 @@
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ */
+
+package com.esri.samples.raster.raster_rendering_rule;
+
+import java.util.List;
+
+import javafx.application.Application;
+import javafx.geometry.Insets;
+import javafx.geometry.Pos;
+import javafx.scene.Scene;
+import javafx.scene.control.ComboBox;
+import javafx.scene.control.Label;
+import javafx.scene.control.ProgressIndicator;
+import javafx.scene.layout.Background;
+import javafx.scene.layout.BackgroundFill;
+import javafx.scene.layout.CornerRadii;
+import javafx.scene.layout.StackPane;
+import javafx.scene.layout.VBox;
+import javafx.scene.paint.Color;
+import javafx.scene.paint.Paint;
+import javafx.scene.text.Font;
+import javafx.stage.Stage;
+import javafx.util.StringConverter;
+
+import com.esri.arcgisruntime.arcgisservices.RenderingRuleInfo;
+import com.esri.arcgisruntime.layers.RasterLayer;
+import com.esri.arcgisruntime.loadable.LoadStatus;
+import com.esri.arcgisruntime.mapping.ArcGISMap;
+import com.esri.arcgisruntime.mapping.Basemap;
+import com.esri.arcgisruntime.mapping.view.MapView;
+import com.esri.arcgisruntime.raster.ImageServiceRaster;
+import com.esri.arcgisruntime.raster.RenderingRule;
+
+public class RasterRenderingRuleSample extends Application {
+
+  private ComboBox<RenderingRuleInfo> renderingRuleInfoComboBox;
+  private MapView mapView;
+  private ArcGISMap map;
+
+  @Override
+  public void start(Stage stage) {
+    try {
+      // create stack pane and application scene
+      StackPane stackPane = new StackPane();
+      Scene scene = new Scene(stackPane);
+
+      // set title, size, and add scene to stage
+      stage.setTitle("Raster Rendering Rule Sample");
+      stage.setWidth(800);
+      stage.setHeight(700);
+      stage.setScene(scene);
+      stage.show();
+
+      // create a control panel
+      VBox controlsVBox = new VBox();
+      controlsVBox.setBackground(new Background(new BackgroundFill(Paint.valueOf("rgba(0,0,0,0.6)"), CornerRadii.EMPTY, Insets.EMPTY)));
+      controlsVBox.setPadding(new Insets(10.0));
+      controlsVBox.setSpacing(8);
+      controlsVBox.setMaxSize(260, 160);
+      controlsVBox.getStyleClass().add("panel-region");
+
+      // create progress indicator
+      ProgressIndicator progressIndicator = new ProgressIndicator();
+      progressIndicator.setVisible(false);
+
+      // create drop down menu of Rendering Rules
+      renderingRuleInfoComboBox = new ComboBox<>();
+      renderingRuleInfoComboBox.setPromptText("Select a Raster Rendering Rule");
```

Comment:
Can remove prompt text since the raster with a rule (albeit "none") is already loaded (see later comment to select "none" from dropdown box initially).